lib/worker/src/domain/vault.rs

```diff
@@ -1,7 +1,7 @@
 use regex::Regex;
 use std::fmt;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I think, if you burrow the `Vault` in this function: `find_cache_config_by_vaults`, you may not need to implement `Clone` here?

Reply:
thanks! yeah, that works. I still tend to just clone to solve the compiler complaint 😅